cypress/fixtures/articles.json

```diff
@@ -3,14 +3,38 @@
     "id": 1,
     "author": "Dash L.",
     "date": "15th of August, 2019",
-    "title": "A study on behaviours of pidgeons",
-    "body": "They are adapting to human lifestyle."
+    "title": "A study on the maladapted social behaviours of pidgeons",
+    "body": "They are seriously mean.",
+    "city": "Stockholm"
   },
   {
     "id": 2,
     "author": "Max A.",
     "date": "13th of August, 2019",
     "title": "The beneficial nature of caffeine",
-    "body": "It's good for your body."
+    "body": "It's good for your body.",
+    "city": "Stockholm"
+  },
+  {
+    "id": 3,
+    "author": "Max A.",
+    "date": "13th of August, 2019",
+    "title": "The beneficial nature of caffeine",
+    "body": "It's good for your body.",
+    "city": "London"
+  },{
+    "id": 4,
+    "author": "Max A.",
```

Comment:
could you make these objects more unique

Reply:
fixed